File: Beginner/Age_in_Days_1020/main.go
package main

import (
	"fmt"
)

func main() {
	var personAgeInDays int
	fmt.Scan(&personAgeInDays)
	years := int(personAgeInDays / 365)
	personAgeInDays = personAgeInDays % 365
	months := int(personAgeInDays / 30)
	personAgeInDays = personAgeInDays % 30

	fmt.Print(years, " ano(s)\n")
	fmt.Print(months, " mes(es)\n")
	fmt.Print(personAgeInDays, " dia(s)\n")
}

File: Beginner/Age_in_Days_1020/Main.java
import java.util.Scanner;

public class Main{
    public static void main(String[] args){
        Scanner sc = new Scanner(System.in);
        int personAgeInDays = sc.nextInt();
        int years = personAgeInDays / 365;
        personAgeInDays = personAgeInDays % 365;
        int months = personAgeInDays / 30;
        personAgeInDays = personAgeInDays % 30;

        System.out.println(years + " ano(s)");
        System.out.println(months + " mes(es)");
        System.out.println(personAgeInDays + " dia(s)");
        sc.close();
    }
}